webapp/views/App/views/Designer/TaxonomyList/Row/useTaxonomyRow.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { matchPath, useLocation, useHistory } from 'react-router'
+
+import * as ObjectUtils from '@core/objectUtils'
+import * as Survey from '@core/survey/survey'
+import * as Taxonomy from '@core/survey/taxonomy'
+
+import { useI18n } from '@webapp/store/system'
+import { useSurveyInfo } from '@webapp/store/survey'
+import { DialogConfirmActions, NotificationActions } from '@webapp/store/ui'
+
+import { appModuleUri, designerModules } from '@webapp/app/appModules'
+
+import * as TaxonomyActions from '@webapp/loggedin/surveyViews/taxonomy/actions'
+
+const useTaxonomyRow = (props) => {
```

Comment:
this is part of the local store. isn't it? move it to store folder and let's use state/Actions convention

Reply:
i will do in the following commit!